gatsby-browser.tsx

```diff
@@ -59,3 +59,8 @@ export const onClientEntry = () => {
     event.preventDefault();
   });
 };
+
+export const onRouteUpdate = ({ location, prevLocation }) => {
```

Comment:
@thecodingwizard is this supposed to print anything to console?

Reply:
yes, though you might need to restart dev server (?)